src/Microsoft.DocAsCode.Dfm/DfmJsonTokenTreeRender.cs

```diff
@@ -0,0 +1,94 @@
+        {
+            return Insert(token, "IncludeBlock");
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, DfmIncludeInlineToken token, MarkdownInlineContext context)
+        {
+            return Insert(token, "IncludeInline");
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, DfmYamlHeaderBlockToken token, MarkdownBlockContext context)
+        {
+            return Insert(token, $"YamlHeader>{Escape(token.Content)}");
+        }
+
+        public override StringBuffer Render(IMarkdownRenderer renderer, MarkdownBlockquoteBlockToken token, MarkdownBlockContext context)
+        {
+            StringBuffer content = StringBuffer.Empty;
+            var splitTokens = DfmBlockquoteHelper.SplitBlockquoteTokens(token.Tokens);
+            foreach (var splitToken in splitTokens)
+            {
+                var sectionToken = splitToken.Token as DfmSectionBlockToken;
+                if (sectionToken != null)
+                {
+                    content += Insert(sectionToken, "SectionBlock");
+                    foreach (var item in splitToken.InnerTokens)
+                    {
+                        content += renderer.Render(item);
+                    }
+                    continue;
+                }
+
+                var noteToken = splitToken.Token as DfmNoteBlockToken;
+                if (noteToken != null)
+                {
+                    var type = noteToken.NoteType.ToUpper();
+                    content += Insert(noteToken, type);
+                    foreach (var item in splitToken.InnerTokens)
+                    {
+                        content += renderer.Render(item);
+                    }
+                    continue;
+                }
+
+                var videoToken = splitToken.Token as DfmVideoBlockToken;
+                if (videoToken != null)
+                {
+                    content += Insert(videoToken, $"VideoBlock>{videoToken.Link}");
+                    continue;
+                }
+
+                foreach (var item in splitToken.InnerTokens)
+                {
+                    content += renderer.Render(item);
+                }
+            }
+            return Insert(token, "BlockquoteBlock", content);
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, DfmFencesBlockToken token, MarkdownBlockContext context)
+        {
+            return Insert(token, "FencesBlock");
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, DfmNoteBlockToken token, MarkdownBlockContext context)
+        {
+            return Insert(token, $"NoteBlock>{Escape(token.Content)}");
+        }
+
+        public virtual StringBuffer Render(IMarkdownRenderer renderer, DfmVideoBlockToken token, MarkdownBlockContext context)
+        {
+            return Insert(token, "VideBlockToken");
```

Comment:
>VideBlockToken [](start = 34, length = 14)

typo, "VideoBlockToken", and I'm thinking should we use nameof(DfmVideoBlockToken) or expose name from IMarkdownToken, to prevent hard code the name in renderer?